sdk/translation/azure-ai-translation-document/tests/test_list_submitted_jobs.py

```diff
@@ -125,7 +125,7 @@ def test_list_submitted_jobs_filter_by_created_after(self, client):
         docs_per_job = 2
 
         # create some jobs
-        start = datetime.now()
+        start = datetime.utcnow()
```

Comment:
was this a bug?

Reply:
This is in the tests, so not an SDK bug. Just an adjustment so that these tests pass for whatever timezone you are running it in.